inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -513,13 +513,19 @@ int main(int argc, char *argv[]) {
         if (!inferRequest) {
             THROW_IE_EXCEPTION << "No idle Infer Requests!";
         }
-
         if (FLAGS_api == "sync") {
             inferRequest->infer();
         } else {
             inferRequest->startAsync();
         }
         inferRequestsQueue.waitAll();
+        auto duration_ms = double_to_string(inferRequestsQueue.getLatencies()[0]);
```

Comment:
does it return read + load + single inference?

Reply:
We report read, load and reshape times and time of first inference.  Full "first inference time" actually depends on real scenario. These 3 (Read, Load, Infer) or 4 (+ Reshape) times can be easily summed up. But I think we should not mislead users doing it yourself.

Reply:
ok, I see